internal/cli/auth/login.go

```diff
@@ -77,14 +77,14 @@ func (opts *loginOpts) SetUpAccess() {
 	}
 	opts.config.Set("service", opts.Service)
 
-	if opts.AuthToken != "" {
-		opts.config.Set("auth_token", opts.AuthToken)
+	if opts.AccessToken != "" {
+		opts.config.Set(config.AccessTokenField, opts.AccessToken)
```

Comment:
oops I broke this, making it a constant to avoid happening again

Reply:
Magic string be gone :D